tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "forceConsistentCasingInFileNames": true,
     "jsx": "react",
     "lib": ["dom", "es2017"],
+    "module": "esnext",
```

Comment:
Is this required or will it also work with other module formats, i.e. `commonjs`?

Reply:
It seems `commonjs` and `esnext` works